pages/checkoutInformationPage.js

```diff
@@ -0,0 +1,31 @@
+export class CheckoutInformationPage {
+  constructor(page) {
+    this.page = page;
+    this.firstNameInput = this.page.locator('[data-test="firstName"]');
+    this.lastNameInput = this.page.locator('[data-test="lastName"]');
+    this.postalCodeInput = this.page.locator('[data-test="postalCode"]');
+    this.continueButton = this.page.locator('[data-test="continue"]');
+  }
+
+  async typeFirstName(firstName) {
+    await this.firstNameInput.type(firstName);
+  }
+
+  async typeLastName(lastName) {
+    await this.lastNameInput.type(lastName);
+  }
+
+  async typePostalCode(postalCode) {
+    await this.postalCodeInput.type(postalCode);
+  }
+
+  async fillInformationData(firstName, lastName, postalCode) {
```

Comment:
Perhaps we could pass the whole user object as a parameter, and then extract the required values in the method?